app/views/review_queues/index.html.erb

```diff
@@ -8,6 +8,7 @@
         <span class="badge"><%= q.items.active.count if q.items.active.count > 0 %></span>
       </h4>
       <p><%= q.description %></p>
+      <%= link_to 'Review History', controller: :review_queues, action: :reviews, name: q.name %>
```

Comment:
Be good to have a named route for this - `review_history_path(name: q.name)`. Use the `:as` key in routes.rb to specify the name.